jobs/v3/auto_complete_sample.go

```diff
@@ -0,0 +1,114 @@
+package cjdsample
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	talent "google.golang.org/api/jobs/v3"
+)
+
+// [START auto_complete_job_title]
+
+/**
+ * Auto completes job titles within given companyName
+ */
+func JobTitleAutoComplete(service *talent.Service, companyName string, query string) (*talent.CompleteQueryResponse, error) {
+	complete := service.Projects.Complete(GetParent()).Query(query).LanguageCode("en-US").Type("JOB_TITLE").PageSize(10)
+	if companyName != "" {
+		complete.CompanyName(companyName)
+	}
+	resp, err := complete.Do()
+	if err != nil {
+		log.Fatalf("Failed to auto complete with query %s in company %s, Err: %v", query, companyName, err)
+	}
+	return resp, err
+
+}
+
+// [END auto_complete_job_title]
+
+// [START auto_complete_default]
+
+/**
+ * Auto completes job titles or company names within given companyName
+ */
+func DefaultAutoComplete(service *talent.Service, companyName string, query string) (*talent.CompleteQueryResponse, error) {
+	complete := service.Projects.Complete(GetParent()).Query(query).LanguageCode("en-US").Type("COMBINED").PageSize(10)
+	if companyName != "" {
+		complete.CompanyName(companyName)
+	}
+	resp, err := complete.Do()
+	if err != nil {
+		log.Fatalf("Failed to auto complete with query %s in company %s, Err: %v", query, companyName, err)
+	}
+	return resp, err
+
+}
+
+// [END auto_complete_default]
+
+// [START auto_complete_sample_entry]
```

Comment:
To be consistent with code samples in other language, entries are like the run-through, and we add test on these run-throughs instead of snippets for simplicity and the dependencies to interact with server.

Reply:
https://github.com/xinyunh0929/java-docs-samples/blob/cloud-talent-solution/jobs/v3/src/main/java/com/google/samples/BasicJobSample.java